kerl

```diff
@@ -1212,40 +1212,35 @@ emit_activate() {
 #!/bin/sh
 # credits to virtualenv
 kerl_deactivate() {
+    set -o allexport
```

Comment:
does this bleed over into the login shell or does this only affect kerl?

Reply:
There's a `set +o allexport` at the end, so unless this breaks in the middle of the execution and we're doing some other concurrent thing (not sure if this is even possible) I'd say it's not relevant. Do you prefer me to add a test? But what kind of test would that be?